test/integration/targets/apt_key/tasks/apt_key_binary.yml

```
@@ -1,9 +1,7 @@
 ---
 
 - name: Ensure import of binary key downloaded using URLs works
-  apt_key: url=https://packages.cloud.google.com/apt/doc/apt-key.gpg
-  # replace the above URL to the following, after the pull request is accepted
-  # apt_key: url=https://github.com/ansible/ansible/tree/devel/test/integration/targets/apt_key/samples/apt-key-example-binary.gpg
+  apt_key: url=https://github.com/ansible/ansible/raw/devel/test/integration/targets/apt_key/samples/apt-key-example-binary.gpg
```

I put this is S3. Please remove `apt_key/samples/apt-key-example-binary.gpg`.
```suggestion
  apt_key:
    url: https://ansible-ci-files.s3.us-east-1.amazonaws.com/test/integration/targets/apt_key/apt-key-example-binary.gpg
```